### PR TITLE
Script correction of NPC ammo_boxes

### PR DESCRIPTION
**Pull request by idk-whoami**
* **Addressed Issue(s)**: 

* **Server Mode**: re

* **Description of Pull Request**:

### CODE REVIEW

File: npc/re/merchants/ammo_boxes.txt [hunk 1]
```diff
@@ -262,7 +262,7 @@ S_Slug_Exchange:
 				case 1:
 					mes "[Kenny]";
 					mes "All done.";
-					delitem .@slug_id, .@new_count * 500;
+					delitem .@slug_id, .@new_count * .@req_count;
```
Comment (Lemongrass3110) [inline npc/re/merchants/ammo_boxes.txt:265]: Should also be done above, where it is output via "mes" or not?
There `.@count` is used directly.

File: npc/re/merchants/ammo_boxes.txt [hunk 2]
```diff
@@ -167,8 +167,8 @@ S_Sphere_Exchange:
 				case 1:
 					mes "[Kenny]";
 					mes "All done.";
-					delitem .@slug_id, .@new_count * 10;
-					getitem .@new_slug_id, .@old_count;
+					delitem .@old_id, .@new_count * 10;
```
Comment (Lemongrass3110) [inline npc/re/merchants/ammo_boxes.txt:170]: Should also be done above, where it is output via "mes" or not?
There `.@old_count` is used directly.

File: npc/re/merchants/ammo_boxes.txt (hunk 1 above)
Comment (idk-whoami) [reply npc/re/merchants/ammo_boxes.txt:265]: if do that, output via "mes" is use .@count countitem, which doesn't match char inventory

what should I do? please advise

**Review by Atemo — APPROVED** [review]
Look good to me!

File: npc/re/merchants/ammo_boxes.txt (hunk 1 above)
Comment (Atemo) [reply npc/re/merchants/ammo_boxes.txt:265]: About that, I don't know how it is on official, I'm in favor of leaving it like that for now

**Pull request merged**